etc/testing/circle/run_tests.sh

```diff
@@ -115,7 +116,7 @@ case "${BUCKET}" in
     make test-enterprise
     # Launch a stand-alone enterprise server in a separate namespace
     make launch-enterprise
-    echo "{\"pachd_address\": \"grpc://${VM_IP}:${ENTERPRISE_PORT}\", \"source\": 2}" | pachctl config set context "enterprise" --overwrite 
```

Comment:
this is just formatting right?

Reply:
yes, I guess my editor did that automatically.